public/app/core/components/AppChrome/MegaMenu/utils.test.ts

```diff
@@ -160,14 +162,6 @@ describe('getActiveItem', () => {
     });
   });
 
-  it('returns the alerting link if the pathname is an alert notification', () => {
-    const mockPathName = '/alerting/notification/foo';
-    expect(getActiveItem(mockNavTree, mockPathName)).toEqual({
-      text: 'Alerting item',
-      url: '/alerting/list',
-    });
-  });
```

Comment:
(I believe) When legacy alerting was removed, we also removed the need for this special case as the route doesn't exist anymore

Reply:
Yep, this was a legacy route so good riddance :)